R/fortify.anosim.R

```diff
@@ -0,0 +1,49 @@
+##' @title Fortify analysis of similarities (ANOSIM) results
+##' @description Prepares fortified versions of results from \code{\link[vegan]{anosim}} objects.
+##' @details TODO
+##' @param model an object of class \code{\link[vegan]{anosim}}.
+##' @param data original data set. Currently ignored.
+##' @param what character; what aspect of the results should be fortified? \code{"ranks"} returns ranks of dissimilarity entries and names of classies. \code{"stats"} returns significance from permutation  and the value of ANOSIM statistic R.
+##' @param ... additional arguments for other methods. Currently ignored.
+##' @return A data frame. For \code{what = "ranks"}, the data frame contains columns \code{'Rank'} and \code{'Class'} with ranks of dissimilarity. For \code{what = "stats"}, the data frame contains columns \code{'Parameter'} and \code{'Value'} with values permutation test and R statistic.
+##'
+##' @export
+##'
+##' @importFrom ggplot2 fortify
+##'
+##' @author Didzis Elferts
+##'
+##' @examples
+##' data(dune)
+##' data(dune.env)
+##' dune.dist <- vegdist(dune)
+##' attach(dune.env)
+##' dune.ano <- anosim(dune.dist, Management)
+##' 
+##' df <- fortify(dune.ano, what = "ranks")
+##' df.title <- fortify(dune.ano, what = "stats")
```

Comment:
Remove this if no strong argument given in response to comments elsewhere in review re having fortify return these kinds of stats.

Reply:
Now fortify() returns just data. P and R values are added only within `autoplot()`.